ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml

```diff
@@ -5,7 +5,7 @@
 
 # Additional steps for RedHat and SLES
 - name: Install cpanminus RedHat 6, CentOS 6 and SLES
-  shell: "curl -L http://cpanmin.us | perl - App::cpanminus"
+  shell: "curl -L https://cpanmin.us | perl - App::cpanminus"
```

Comment:
Hopefully curl on all our boxes is SSL-enabled :-)

Reply:
If they don't then we've got bigger problems :-)